tests/foreman/api/test_role.py

```diff
@@ -100,7 +100,48 @@ def test_positive_update(self):
 class CannedRoleTestCases(APITestCase):
     """Implements Canned Roles tests from API"""
 
-    @stubbed()
+    def create_org_admin(self, name=None, orgs=None, locs=None):
```

Comment:
nitpick: personally my first guess when reading `create_org_admin` is about admin user, not admin role. Maybe it's better to rename to `create_org_admin_role` to avoid confusion?